build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
this change is because, when running in a manylinux container, `python3` picks up some random python 3.9 installation, rather than the intended python 3.10 or so python.